tests/save-and-return.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava'
+import withPage from '../utils/withPage'
+import config from '../config'
+
+test(
+  'Save & return button is displayed to the user',
+  withPage,
+  async (t, page) => {
+    await page.goto(config.formURL)
+    t.is(
+      await page.getText('a[href=\'/return\']'),
```

Comment:
Can we make the sign-in link be a variable created in config?

Reply:
Still a WIP